client/store/modules/editor/getters.js

```diff
@@ -21,8 +21,8 @@ export const contentContainers = (_state, getters, { repository }) => {
 };
 
 export const assessments = (_state, getters, rootState) => {
-  const { items: tes } = rootState.repository.tes;
```

Comment:
Since this is now custom container should there be a separate getter for it at all ?

Reply:
No there should not. I'll remove it after I merge with develop.